src/styles/_calendar.scss

```diff
@@ -0,0 +1,49 @@
+.calendar {
+  margin: auto;
+  display: flex;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  width: $calendarWidth;
+  gap: 1px;
+  padding-top: 50px;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 30px;
+    font-family: Arial, sans-serif;
+    background-color: #eee;
+    border: 1px solid #000;
+    box-sizing: border-box;
```

Comment:
move it to the `*` selector into normalize file
```suggestion

```

Reply:
ok